operations/local-replication/README.md

```diff
@@ -0,0 +1,499 @@
+         "mode":"secondary",
+         "primary_cluster_addr":"https://127.0.0.1:8201",
+         "secondary_id":"vault2",
+         "state":"stream-wals"
+      }
+   },
+   ...
+}
+
+# OPTION 2 - Demotion of replication role 
+# demote primary to secondary
+vault write -f /sys/replication/performance/primary/demote
+
+## demoting dr primary to secondary puts it in cold standby
+# vault write -f /sys/replication/dr/primary/demote
+
+
+# check performance secondary for access to secrets etc
+vault2 auth -method=userpass username=drtest password=drtest
+vault2 read supersecret/drtest
+
+# note that the .vault-token has changed
+diff ~/.vault-token ~/.vault-token-DRTEST
+
+## Promote DR secondary to primary
+vault3 write -f /sys/replication/dr/secondary/promote dr_operation_token=${DR_OPERATION_TOKEN}
+
+# check status
+vault3 read -format=json sys/replication/status
+
+# let's check our token status
+cp ~/.vault-token-DRTEST ~/.vault-token
+vault3 read supersecret/drtest
+
+# vault3 read supersecret/drtest
+## Key             	Value
+## ---             	-----
+## refresh_interval	768h0m0s
+## password        	baines
+## username        	harold
+
+## SUCCESS!
+
+```
+
+The environment looks like the following at this step:
+
+```
++---------------------------------+                    +------------------------------------+
+| vault port:8200                 |                    | vault2 port:8202                   |
+| Replication disabled            |                    | Performance secondary replication  |
+| (or demoted)                    |                    | vault3 --> vault2                  |
+|                                 |                    |                                    |
++---------------------------------+                    +------------------------------------+
+
+                                                                          ^
+                                                                          |
+                                                                          |
+                                                                          |
++---------------------------------+                                       |
+| vault3 port:8204                |                                       |
+| DR primary replication          |  +------------------------------------+
+| Performance primary replication |
+| vault3 --> vault2               |
++---------------------------------+
+```
+
+
+FAILBACK - Option 1  - relevant for Vault 0.8-0.9
+Note that this is not an idea situation today, as we must first sync DR replication set back to vault, then perform another failover such that vault is the perf primary/dr primary.
+```
+# Disable replication on vault (if not already done)
+vault write -f /sys/replication/dr/primary/disable
+vault write -f /sys/replication/performance/primary/disable
+
+# enable vault as DR secondary to vault3
+vault3 auth root
+vault3 write -f /sys/replication/dr/primary/enable
```

Comment:
Is this step necessary? IIRC the cluster should already be a dr primary, could be wrong though